information-system/data-store/access-control.md

```diff
@@ -0,0 +1,101 @@
+parent: Data store
+---
+
+{: .no_toc }
+
+<details  open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+* TOC
+{:toc}
+____
+</details>
+
+# Access Control Overview
+
+This document will discuss the types of access controls available and then reveal the methods to apply these access controls through the Datastore User Interface. 
+
+## What access controls are available for datastore datasets?
+ 
+There are three access rights to consider: 
+
+1. Metadata access
+2. Data access
+3. Admin access
+
+
+## Metadata access
+
+Facilitates control of general viewing and metadata updating.
+
+### Metadata Read
+
+- If enabled, users with this right will be able view the dataset's metadata and find the dataset in the datastore "search" and "list" tools. 
+- If disabled, users will not be able to find the dataset in the datastore let alone view the metadata.
+
+
+### Metadata Write
+
+- If enabled, users can update metadata fields and will be granted metadata read access regardless of the setting for metadata read.
+- If disabled, general users will not be able to update the metadata.
+
+## Data access
+
+Facilitates control over the upload/download of data to/from the storage location associated with the dataset.
+
+### Data read
+
+- If enabled, the download panel will become visible to users and they will be able to download data from the storage location. 
+- If disabled, the download panel will not be visible and users will not be able to download data from the storage location.
+
+### Data write
+
+- If enabled, users will be able to upload/overwrite and download to/from the storage location as both the upload and download panels will be made available to general users.
+- If disabled, users will not be able to upload/overwrite data in the storage location.
+
+## Admin access
+
+- If enabled, provides administrative control over the dataset. That is, the ability to lock and unlock the dataset in addition to all of the above privileges. For more information on locking and unlocking, see here.
+
+# Applying Access Controls
+
+The aforementioned controls can be applied for individual datasets by those who have administration access to the dataset. By default, this is the owner of the dataset and Information System Admins.
+
+Each dataset can have a rule for its accessibility to (the group of all) general datastore users, and (optionally) extra rules for multiple custom groups of users. The following sections will discuss these.  
+
+## General user access to dataset
+
+### Who are general users?
+
+General Datastore users refers to the group of all users who can access the datastore. As such, general user access controls facilitate the general privacy or publicity (to all datastore users) of a dataset. You could limit access controls for general users if you wish to keep your dataset private to a group (perhaps during development processes and then later release it by updating general user access rights).
```

Comment:
Prefer "Data Store" over datastore consistently

Reply:
This is a bit wordy

Maybe

> The "General Users" group contains all users of the Data Store. Each dataset must specify a set of access grants for this group. This access could range from full privacy (no access grants), to complete control (data write).

There is an edge case here which is that the limiting factor for a user is still their overall permissions in the data store - not sure where that fits in.

Reply:
Resolved